src/main/java/uk/gov/mca/beacons/api/jpa/entities/BeaconNote.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.mca.beacons.api.jpa.entities;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.Email;
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.mca.beacons.api.domain.NoteType;
+
+@Entity
+@Getter
+@Setter
+public class BeaconNote {
+
+  @Id
+  @GeneratedValue
+  private UUID id;
+
+  private UUID beaconId;
+
+  private String note;
+
+  @Enumerated(EnumType.STRING)
+  private NoteType type;
+
+  private LocalDateTime createdDate;
+
+  private UUID personId;
+
+  private String fullName;
+
+  @Email
```

Comment:
@Ninamma I would probably remove this annotation from the JPA entity.  It's used for validation to check if the email address is valid and Hibernate will fail to persist it if it not a valid one.

For the migration we may have users without a valid email address so I would be tempted to say we remove this from here and add validation in the service layer if we need it?